app/views/posts/index.html.erb

```diff
@@ -1 +1,36 @@
-<h1>List of all posts by user </h1>
\ No newline at end of file
+
+<div>
+
+   <div class=" d-flex mb-2 m-5 text-center" style="max-width: 1140px;">
+        <div class="col-md-4">
+          <img src="<%= @user.photo %>" class="img-fluid rounded-start" >
```

Comment:
- [ ] Lastly, please fix the photo also because currently, It looks broken 🙏 
![image](https://user-images.githubusercontent.com/90222110/222829801-2ea6d4b3-f035-47c8-b92a-6aed2c8a0028.png)

Reply:
- Great job fixing the image rendering error 👍🏽 @Suzi216 

<img width="1326" alt="Screenshot 2023-03-04 at 13 35 22" src="https://user-images.githubusercontent.com/58692590/222901408-3ca62ce3-6a9b-44d7-8acb-6ea7b9e40f6c.png">